src/main/java/de/focusshift/zeiterfassung/absence/AbsenceWriteEntity.java

```diff
@@ -141,9 +141,12 @@ public int hashCode() {
 
     @Override
     public String toString() {
-        return "AbsenceEntity{" +
-            "tenantId=" + tenantId +
-            "id=" + id +
-            '}';
+        return "AbsenceWriteEntity{" +
+                "tenantId='" + tenantId + '\'' +
+                ", sourceId=" + sourceId +
+                ", userId='" + userId + '\'' +
```

Comment:
die `userId` würde ich eher rausnehmen, damit da keine Verbindung zum user hergestellt werden kann aus den logs. mit der `id` können wir das intern in der Datenbank nachschauen und nachvollziehen.